lib/lha_arch_amiga.c

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
I think if you add the name of this file to the `SRC =` list in `lib/Makefile.am` that will probably get you ~most of the way to a working autotools compile.

Reply:
Done!